src/Objs/Homepage/HomepageEditingConfig.ts

```diff
@@ -110,5 +124,37 @@ provideEditingConfig(Homepage, {
     siteFontHeadlineWeight: '500',
     siteRoundedCorners: true,
   },
-  validations: defaultPageValidations,
+  validations: [
```

Comment:
* [ ] FYI all this validation code could go if we pick a font name instead of making it editable. Win-win?

Reply:
If we no longer explicitly set a font-name in each font obj we run the risk, that an editor might accidentally select different fonts in the `referencelist`. With a shared font name this risk can be minimized.

Reply:
I don't see a risk. For demos and prototypes a single weight will be used anyway. For prod use, a dynamic font is recommended for page speed. This leaves us with only exceptional cases that use multiple fonts at all.